<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageMPSettingsSocial</class>
 <widget class="QWidget" name="PageMPSettingsSocial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>315</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="1">
    <widget class="QComboBox" name="fieldTeamChanging">
     <item>
      <property name="text">
       <string>Disabled</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Enabled</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Balancing Only</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldObservers">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Enable Observers</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelTeamChanging">
     <property name="text">
      <string>Team Changing</string>
     </property>
     <property name="buddy">
      <cstring>fieldTeamChanging</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldFriendlyFire">
     <property name="text">
      <string>Friendly Fire</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldBetrayalBooting">
     <property name="text">
      <string>Betrayal Booting</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="fieldDeadVoice">
     <property name="text">
      <string>Allow Dead Players to Talk</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldGlobalVoice">
     <property name="text">
      <string>Don't Team-Restrict Voice Chat</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldProximityVoice">
     <property name="toolTip">
      <string>If enabled, players will be able to hear voice chat from any physically nearby enemies during team games.</string>
     </property>
     <property name="text">
      <string>Proximity Voice</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fieldObservers</tabstop>
  <tabstop>fieldTeamChanging</tabstop>
  <tabstop>fieldFriendlyFire</tabstop>
  <tabstop>fieldBetrayalBooting</tabstop>
  <tabstop>fieldProximityVoice</tabstop>
  <tabstop>fieldGlobalVoice</tabstop>
  <tabstop>fieldDeadVoice</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
